test/cyclonus/install-cyclonus.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cyclonus
+  namespace: kube-system
+  labels:
+    app: cyclonus
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - command:
+            - ./cyclonus
+            - generate
+            - --noisy=true
+            - --ignore-loopback=true
+            - --cleanup-namespaces=true
+            - --perturbation-wait-seconds=5
+            - --pod-creation-timeout-seconds=20
+          name: cyclonus
+          imagePullPolicy: IfNotPresent
+          image: mfenwick100/cyclonus:v0.4.4
+
```

Comment:
reason for newline here?

Reply:
It seems it will be nice to have flexibility to control command later to have various testing environment.